recipe/meta.yaml

```
@@ -10,31 +10,33 @@ source:
   sha256: fdc2739627eb1047c6b2df35045b654e07924fc9f6f8d490a34967365e75fd60
 
 build:
+  number: 1
   noarch: python
-  number: 0
   script:
     - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

These flags are [already set by default when building Conda recipes]( https://github.com/conda/conda-build/blob/a84ac36176e23329625e0badc265e9c434a44c60/conda_build/build.py#L3024-L3044 ). So we can simplify this by dropping them

```suggestion
    - {{ PYTHON }} -m pip install . -vvv
```